web3/inpage.js

```diff
@@ -174,14 +49,11 @@ if (!window.chrome || !window.chrome.extension) {
 
       // Else, add a full web3 instance
       if (!web3.injectedWeb3) {
-        const rawWeb3 = require('web3/dist/web3.min.js');
-        eval(rawWeb3); // eslint-disable-line no-eval
-
-        const injectedWeb3 = new window.Web3(web3.currentProvider);
-        web3.injectedWeb3 = injectedWeb3;
-        // Expose object directly
-        window.web3 = injectedWeb3;
-        return injectedWeb3[name];
+        require('./lib');
```

Comment:
I think for now we should just inject `web3` (instance) and `Web3` (class) to every page without a Proxy, and then allow to configure the behaviour when #55 is implemtened.